src/index.html

```diff
@@ -2,15 +2,36 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper-big" data-qa="big">
+      <input
+        class="search-field-style big-search-field"
+        type="search"
+        name="big-search-bar"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
+      <img
+        class="big-search-icon"
+        src="./images/Search.svg"
+        alt="search icon">
```

Comment:
don't use such words in alt like image, icon, etc.
screenreaders know this is an image and say search icon image in you case